### PR TITLE
Bundle all SASS in screen.scss

### PR DESCRIPTION
**Pull request by OmeGak**
Missing:
- [x] Adapt once #2758 is merged
- [x] Make sure this doesn't break meeting themes
- [x] Test thoroughly (visit as many indico pages as possible to see if there are CSS bugs)

### CODE REVIEW

File: indico/MaKaC/webinterface/pages/contributionReviewing.py [hunk 1]
```diff
@@ -123,12 +123,6 @@ def _getHeadContent(self):
                '\n'.join(['<script src="{0}" type="text/javascript"></script>'.format(url)
                           for url in self._asset_env['mathjax_js'].urls()])
 
-    def getCSSFiles(self):
-        return WPConferenceModifBase.getCSSFiles(self) + self._asset_env['contributions_sass'].urls()
-
-    def getJSFiles(self):
```
Comment (ThiefMaster) [inline indico/MaKaC/webinterface/pages/contributionReviewing.py]: this one isn't needed?

**Comment by nop33** [comment]
The `.big` class in `.i-button` seems to be broken:

image

**Comment by pferreir** [comment]
> The .big class in .i-button seems to be broken:

Now that's BIG.

**Comment by nop33** [comment]
Something to take into account: The rules in `jquery-ui-custom.css` sometimes have higher specificity than the `screen.scss`. An example is the color of the links in dialogs, where we have
``` css
.ui-widget-content a {
    color: #4C6A83;
}
```
but in the `_links.scss` we have
``` scss
a {
    color: $blue;
    ...
}
```

File: indico/MaKaC/webinterface/pages/contributionReviewing.py (hunk 1 above)
Comment (OmeGak) [inline indico/MaKaC/webinterface/pages/contributionReviewing.py]: Why would it be? The point is for `contributions_sass` to be included in `screen.scss`, so we don't need to serve extra bundles through `getCSSFiles`. Am I missing anything?

**Comment by OmeGak** [comment]
The big buttons are now fixed.

**Comment by OmeGak** [comment]
The specifity problems should now be fixed. I think it only affected links within dialogs thanks to annoying super-intrusive jQuery UI theme.

File: indico/web/assets/bundles.py [hunk 2]
```diff
@@ -422,43 +422,10 @@ def rjs_bundle(name, *files):
...
-news_sass = sass_module_bundle('news')
-users_sass = sass_module_bundle('users')
-auth_sass = sass_module_bundle('auth')
-surveys_sass = sass_module_bundle('surveys')
-timetable_sass = sass_module_bundle('timetable')
-sessions_sass = sass_module_bundle('sessions')
-markdown_sass = sass_module_bundle('markdown')
+                     "sass/partials/*.scss",
+                     "sass/modules/*.scss",
+                     "sass/modules/*/*.scss"]
```
Comment (ThiefMaster) [inline indico/web/assets/bundles.py]: `sass/modules/**/*.scss` might cover both this and the previous one (`**` matches within the directory and any subdirectory). At least if webassets supports `**`

**Pull request merged**